cnv_germline_case_scattered_workflow.inputs.json

```diff
@@ -0,0 +1,37 @@
+{
+  "CNVGermlineCaseScatteredWorkflow.normal_bams": [
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74NEG_20xy-downsampled.bam",
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74P2T_20xy-downsampled.bam",
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74P35_20xy-downsampled.bam"
+    ],
+  "CNVGermlineCaseScatteredWorkflow.normal_bais": [
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74NEG_20xy-downsampled.bam.bai",
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74P2T_20xy-downsampled.bam.bai",
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74P35_20xy-downsampled.bam.bai"],
+  "CNVGermlineCaseScatteredWorkflow.contig_ploidy_model_tar": "gs://gatk-best-practices/cnv_germline_pipeline/wes-do-gc-contig-ploidy-model.tar.gz",
+  "CNVGermlineCaseScatteredWorkflow.gatk_docker": "broadinstitute/gatk:4.1.5.0",
+  "CNVGermlineCaseScatteredWorkflow.allosomal_contigs": ["X", "Y"],
+  "CNVGermlineCaseScatteredWorkflow.ref_copy_number_autosomal_contigs": 2,
+  "CNVGermlineCaseScatteredWorkflow.gcnv_disable_annealing": true,
+  "CNVGermlineCaseScatteredWorkflow.gcnv_initial_temperature": 1.0,
+  "CNVGermlineCaseScatteredWorkflow.gcnv_max_training_epochs": 1,
+  "CNVGermlineCaseScatteredWorkflow.gcnv_min_training_epochs": 1,
+  "CNVGermlineCaseScatteredWorkflow.gcnv_model_tars": [
+    "gs://gatk-best-practices/cnv_germline_pipeline/wes-do-gc-gcnv-model-0.tar.gz",
+    "gs://gatk-best-practices/cnv_germline_pipeline/wes-do-gc-gcnv-model-1.tar.gz"],
+  "CNVGermlineCaseScatteredWorkflow.gcnv_num_thermal_advi_iters": 1,
+  "CNVGermlineCaseScatteredWorkflow.intervals": "gs://gatk-best-practices/cnv_germline_pipeline/ice_targets_chr20xy.interval_list",
+  "CNVGermlineCaseScatteredWorkflow.filtered_intervals": "gs://gatk-best-practices/cnv_germline_pipeline/ice_targets_chr20xy.preprocessed.filtered.interval_list",
+  "CNVGermlineCaseScatteredWorkflow.num_intervals_per_scatter": 20,
+  "CNVGermlineCaseScatteredWorkflow.num_samples_per_scatter_block": 2,
+  "CNVGermlineCaseScatteredWorkflow.gcnv_max_advi_iter_first_epoch": 10,
+  "CNVGermlineCaseScatteredWorkflow.gcnv_log_emission_sampling_rounds": 1,
+  "CNVGermlineCaseScatteredWorkflow.gcnv_log_emission_samples_per_round": 1,
+  "CNVGermlineCaseScatteredWorkflow.gcnv_max_advi_iter_subsequent_epochs": 1,
+  "CNVGermlineCaseScatteredWorkflow.gcnv_max_copy_number": 3,
+  "CNVGermlineCaseScatteredWorkflow.gcnv_max_calling_iters": 1,
+  "CNVGermlineCaseScatteredWorkflow.maximum_number_events_per_sample": 120,
+  "CNVGermlineCaseScatteredWorkflow.ref_fasta_dict": "gs://gatk-best-practices/cnv_germline_pipeline/Homo_sapiens_assembly19.truncated.dict",
```

Comment:
Do we still want to use hg19 test data?

Reply:
Good point.  I say onwards and upwards -- to hg38!

Reply:
Happy to use another set but I'll need help finding resource files (e.g. contig_ploidy_model_tar)

Reply:
Once we decide on the appropriate sample BAMs, the resource models used in the case workflow will be generated by running the cohort workflow.

Reply:
If its helpful here are a list of downsample 1k human genome project bam samples files here: https://console.cloud.google.com/storage/browser/gatk-test-data/1kgp/downsampled_bam_hg38/?project=broad-dsde-outreach&organizationId=548622027621